chocolatey/docs
# (#1088) Upgrade choco-astro to 0.1.2

## Description Of Changes
This upgrades choco-astro to the latest version, 0.1.2. This does not bring in any features or functionality changes.

## Motivation and Context
A new version of choco-astro  has been released, and it needs updated here.

## Testing


This was ran locally on my Windows 10 instance running on Chrome.

1. Run the site
2. Click around, make sure everything is "the same". No functionality or features have been added.

## Change Types Made
    * Issue #

## Change Checklist


## Related Issue
* https://github.com/chocolatey/docs/issues/1088
* https://github.com/chocolatey/choco-astro/issues/9

Fixes #1088
